scripts/scr_event_dudes/scr_event_dudes.gml

```diff
@@ -8,14 +8,14 @@ function scr_event_dudes(do_action, is_planet, system_name, location_id) {
 	*/
 
 	if (do_action=1){
-	    if (obj_ini.progenitor=0){
+	    if (obj_ini.progenitor == PROGENITOR.DARK_ANGELS) {
```

Comment:
Here, I'm not sure, but I think it may be the same case, where 0 means none or random(?).

Reply:
This is changed to `PROGENITOR.NONE` in [184fe4f](https://github.com/EttyKitty/ChapterMaster/pull/44/commits/184fe4fcf3b2304a2d8baedc0847da80dd1509fc)